### ISSUE TITLE
Request: Fix Plugin and Publish to Forum

### ISSUE THREAD

**Issue by Cobwebster** [BedrockCommunity/CoreJobs-Nukkit#1]
Could this be put on nukkit resources? https://nukkitx.com/resources/ It would hit more people that way.

**Comment by ghost** [comment]
Yes, but anyone test it first? I want to know if works or not. I can't now because i work on SimpleMessages-Nemisys.

I will add and more features to this too coming soon.

**Comment by ghost** [comment]
Tested and this is message: Not work for now. Client crash when you use /job. Need figure out why. On my core works perfectly, but i don't use config.
